pkg/logger/event/README.md

```diff
@@ -108,3 +108,14 @@ event.AdditionalFields(req *restful.Request, fields map[string]interface{})
 ```
 
 Pay attention on the field key name not to overwrite the existing default fields.
+
+### Timestamp format
+
+By importing this package, it will set the logrus timestamp for entire service to use RFC3339 in millisecond precision 
+and will force the timezone to be UTC. This timestamp format is required for the AccelByte Event Log service.
+
+However if you want to use your own timestamp format, you can override it by calling `logrus.SetFormatter` in your code.
+This is the example of setting timestamp format to use `RFC3339Nano`.
+```go
+logrus.SetFormatter(&logrus.TextFormatter{TimestampFormat: time.RFC3339Nano})
```

Comment:
Where should I place this code if want to change the default?

Reply:
In the main function. I'll add this to the readme. Thanks!